main/src/library/DelayList.flix

```diff
@@ -1169,6 +1169,13 @@ namespace DelayList {
         };
         Iterator(done, next)
 
+    ///
+    /// Returns an iterator over `l` zipped with the indices of the elements,
+    /// i.e. an alias for `toIterator(r, l) |> Iterator.zipWithIndex`.
```

Comment:
Lets not reveal the implementation details. I think that's bad.

Reply:
Alrighty :)